decidim-verifications/app/views/decidim/verifications/id_documents/authorizations/_form.html.erb

```diff
@@ -1,13 +1,9 @@
 <%= form_required_explanation %>
 
-<div class="card-section">
-  <div class="field">
-    <%= form.select :document_type, @form.document_types_for_select, prompt: true %>
-  </div>
+<div class="form__wrapper">
+  <%= form.select :document_type, @form.document_types_for_select, prompt: true %>
 
-  <div class="field">
-    <%= form.text_field :document_number %>
-  </div>
+  <%= form.text_field :document_number %>
 
   <% if using_online? %>
     <%= form.upload :verification_attachment, button_class: "button button__lg button__transparent-secondary verification__upload-button" %>
```

Comment:
The upload modal looks weirdly aligned on my local, different than in the staging app:
![image](https://user-images.githubusercontent.com/864340/203937637-2f693dd5-d8f8-4471-a1c4-0a19d0c14cbf.png)

I thought that this issue would be resolved by merging the upload modal PR.

Is this something that will be fixed in another PR?

Reply:
The modals CSS markup has been refactorized in some other branch, to make them responsive. This branch does not contain such changes, but you can go to [staging](https://decidim-redesign.populate.tools/id_documents/authorizations/edit) to double check it. CSS issues you notify in a branch, go to staging to see whether is really fixed or not. Staging prevails (you know, one ~ring~ branch to rule them all...)